src/ray/raylet/placement_group_resource_manager.cc

```diff
@@ -160,6 +185,11 @@ void NewPlacementGroupResourceManager::ReturnBundle(
     }
   }
   pg_bundles_.erase(it);
+  // Erase from `bundle_spec_map_`.
```

Comment:
oh is it a bug from existing code? nice discovyer!

Reply:
Yes!